src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,99 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize--xs: $pf-global--FontSize--xs; // Axis Labels
+$pf-chart-global--FontSize--sm: $pf-global--FontSize--sm; // For Legend
+$pf-chart-global--FontSize--2xl: $pf-global--FontSize--2xl; // For Large Numbers
```

Comment:
Do we want to have a var for the chart font family? This is currently hard coded as global_FontFamily_sans_serif.value

Reply:
I tagged @mcoker about this as I'm not sure the best way to import the new font family

Reply:
I think so. Seems like we would want the chart theme to only use `$pf-chart` vars? Then it allows us the flexibility in the future to deviate from core by simply updating the variable value, instead of also needing to update the react chart theme code.